pom.xml

```diff
@@ -103,6 +103,33 @@
             <version>2.2.9</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>1.6.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito-common</artifactId>
+            <version>1.6.5</version>
+        </dependency>
```

Comment:
powermock-api-mockito-common is not required, there is a PR to change this.

Reply:
Done, thanks.